MachUpskillingFY17/JabbR-Core
# Justine/account register

To Test:

- Run the project
- Change the url to be http://localhost:{PORT}/account/register
- Validate form error messages
    - enter any username
    - enter any correctly formatted email
    - enter a password with less than 6 characters and a confirmation password that doesn't match
    - press Register button
        - Should see validation error messages at the bottom of the page
- Enter correct form validation
    - enter any username
    - enter any correctly formatted email
    - enter a password with at least 6 characters, a number and a non-alphanumeric character and a confirmation password that matches
    - press Register button
        - JabbR home page should load and function as expected (create room, join room, message in room etc.) 
- Verify the user was created
    - In the SQL Server Object Explorer open the JabbRCoreDB 
    - Right click on the "ASPNetUsers" table and select "View Data"
        - Should see the user you just created in the table
